lib/active_merchant/billing/gateways/credorax.rb

```diff
@@ -405,8 +407,10 @@ def commit(action, params, reference_action = nil)
 
       def sign_request(params)
         params = params.sort
-        params.each { |param| param[1].gsub!(/[<>()\\]/, ' ') }
-        values = params.map { |param| param[1].strip }
+        values = params.map do |param|
+          value = param[1].gsub(/[<>()\\]/, ' ')
+          value.strip
+        end
```

Comment:
this change is totally separate from the 3DS related update, correct?

Reply:
Yeah, it's to fix an error in tests (and application code if an developer were to freeze attribute values). You'll see the failures if you run unit tests for Credorax on master.